Objective-C/CBLDatabase.h

```diff
@@ -111,7 +111,7 @@ __deprecated_msg("Use [database defaultCollection].count instead.");
  @return The CBLDocument object.
  */
 - (nullable CBLDocument*) documentWithID: (NSString*)id
-__deprecated_msg("Use [database defaultCollection] documentWithID:] instead.");
+__deprecated_msg("Use [database defaultCollection] documentWithID:] instead.") NS_SWIFT_NOTHROW;
```

Comment:
Does it need here as well when no out error?

Reply:
its not necessary, I might have accidentally updated with collection documentWithID:error: